fmf/base.py

```diff
@@ -365,6 +367,11 @@ class describing the environment context. By default, the key
         Optional 'decision_callback' callback would be called for every adjust
         rule inspected, with three arguments: current fmf node, current
         adjust rule, and whether it was applied or not.
+
+        Optional 'additional_rules' parameter can be used to specify rules
+        which should be applied in addition to those defined in the node itself.
+        These additional rules are processed even when the applied
+        rule defined in the data has ``continue: false`` set.
```

Comment:
About the `continue` comment, isn't that contradicting with the `if not continue_: break`?

Reply:
Yeah, it seems that if any of the node rules has `continue: false` the loop is ended an thus additional rules would not be processed.

Reply:
good catch. 'continue: false' needs to be bound to the 'node' or 'additional_rules'

Reply:
rationale: 'additional_rules' comes from runtime and they _have to be processed regardless 'continue' value in the rules from node metadata.

Reply:
For me `additional` reads as append, probably because of the relation to `+`. Was hoping that `+<` had a documented naming to use for the alternative, but no luck.

Had to detangle the concepts of rule and grammar in my head since this seems to address the former. So the usage of it is like adding dict key-values to the `adjust` (although we have the issue of inconsistent operation order) in which case, similar to inheritance it feels like it should be appended. (For the grammar part of the operations, it shouldn't really matter, which is why I got confused for a second)

Reply:
Described in 886fcdb

Reply:
The only motivation for this is to make https://github.com/teemtee/tmt/discussions/2430.

Reply:
Thanks for the clarification and docs enhancement.